### ISSUE TITLE
Archivematica - Quick-Start Guide

### ISSUE THREAD

**Issue by hstgelais-crkn-rcdr** [crkn-rcdr/Digital-Preservation#24]
https://www.archivematica.org/en/docs/archivematica-1.13/getting-started/quick-start/quick-start/#installing-on-a-virtual-machine-using-vagrant

* Try to create an AIP using their demo install
* Try to create an AIP using the new test installation from above.
* Follow each of the tasks, which will include downloading an AIP and looking at the contents with 7zip

**Comment by hstgelais-crkn-rcdr** [comment]
Sub task of #16

**Comment by hstgelais-crkn-rcdr** [comment]
I tried to install Archivematica on the new Openstack server and it failed. I created a ticket on their support forum [here](https://groups.google.com/g/archivematica/c/_eKroj-dcxQ). 

I am waiting for Archivematica comes back to us with a resolution? support?

**Comment by RussellMcOrmond** [comment]
In the interim it may be useful for you to do some of your own test transfers using their sample data and the Vagrant you already had running in the past.  Worry about the new one later.


What is included in the package comes from https://github.com/artefactual/archivematica-sampledata

I pointed a transfer location to /home/vagrant/archivematica-sampledata  on my test machine (I believe the default has /home , which didn't work for me to do transfers).  I've since tried to upgrade it, which broke because of that dependency issue.

In the past (Apparently August 2017) I created a sub-directory on the NAS server called  "Archivematica" which I used for test transfers, which is now  /media/archivematica/Archivematica/ inside the currently running Vagrant VM.

**Comment by RussellMcOrmond** [comment]
The update in https://github.com/artefactual/archivematica/pull/1780 seems to have solved the problem with building a fresh Vagrant install.

```
archivematica@westbengal:~/archivematica-1.13.0/deploy-pub/playbooks/archivematica-bionic$ vagrant halt
==> am-local: Attempting graceful shutdown of VM...
archivematica@westbengal:~/archivematica-1.13.0/deploy-pub/playbooks/archivematica-bionic$ vagrant destroy am-local
    am-local: Are you sure you want to destroy the 'am-local' VM? [y/N] y
==> am-local: Destroying VM and associated drives...
archivematica@westbengal:~/archivematica-1.13.0/deploy-pub/playbooks/archivematica-bionic$ vagrant up
```

The first time I ran it there was a timeout setting up the pipeline.

`TASK [artefactual.archivematica-src : Create Dashboard user and register pipeline on SS] ***`

I tried `vagrant partition`, but that didn't set up the pipeline -- it just said everything was a success.

```
PLAY RECAP *********************************************************************
am-local                   : ok=210  changed=43   unreachable=0    failed=0    skipped=219  rescued=0    ignored=0   
```



Trying again, but this is a totally separate problem that will have a different solution.

**Comment by RussellMcOrmond** [comment]
Ahh.. So same error the second time.

```
TASK [artefactual.archivematica-src : Create Dashboard user and register pipeline on SS] ***
fatal: [am-local]: FAILED! => {"changed": false, "cmd": "./manage.py install  --username=\"admin\" --password=\"archivematica\" --email=\"admin@example.com\" --site-url=\"http://192.168.168.198\" --org-name=\"test org\" --org-id=\"test id\" --api-key=\"this_is_the_am_api_key\" --ss-url=\"http://192.168.168.198:8000\" --ss-user=\"admin\" --ss-api-key=\"this_is_the_ss_api_key\" --whitelist=\"\"\"\"\n --pythonpath=/usr/lib/archivematica/archivematicaCommon", "msg": "\n:stderr: Traceback (most recent call last):\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connection.py\", line 170, in _new_conn\n    (self._dns_host, self.port), self.timeout, **extra_kw\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/util/connection.py\", line 96, in create_connection\n    raise err\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/util/connection.py\", line 86, in create_connection\n    sock.connect(sa)\nsocket.timeout: timed out\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 706, in urlopen\n    chunked=chunked,\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 394, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connection.py\", line 234, in request\n    super(HTTPConnection, self).request(method, url, body=body, headers=headers)\n  File \"/usr/lib/python3.6/http/client.py\", line 1281, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"/usr/lib/python3.6/http/client.py\", line 1327, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"/usr/lib/python3.6/http/client.py\", line 1276, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"/usr/lib/python3.6/http/client.py\", line 1042, in _send_output\n    self.send(msg)\n  File \"/usr/lib/python3.6/http/client.py\", line 980, in send\n    self.connect()\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connection.py\", line 200, in connect\n    conn = self._new_conn()\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connection.py\", line 177, in _new_conn\n    % (self.host, self.timeout),\nurllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPConnection object at 0x7f241b4b4a90>, 'Connection to 192.168.168.198 timed out. (connect timeout=20.0)')\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/requests/adapters.py\", line 449, in send\n    timeout=timeout\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 756, in urlopen\n    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/urllib3/util/retry.py\", line 574, in increment\n    raise MaxRetryError(_pool, url, error or ResponseError(cause))\nurllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='192.168.168.198', port=8000): Max retries exceeded with url: /api/v2/pipeline/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f241b4b4a90>, 'Connection to 192.168.168.198 timed out. (connect timeout=20.0)'))\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"./manage.py\", line 13, in <module>\n    execute_from_command_line(sys.argv)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/django/core/management/__init__.py\", line 364, in execute_from_command_line\n    utility.execute()\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/django/core/management/__init__.py\", line 356, in execute\n    self.fetch_command(subcommand).run_from_argv(self.argv)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/django/core/management/base.py\", line 283, in run_from_argv\n    self.execute(*args, **cmd_options)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/django/core/management/base.py\", line 330, in execute\n    output = self.handle(*args, **options)\n  File \"/vagrant/src/archivematica/src/dashboard/src/installer/management/commands/install.py\", line 74, in handle\n    setup_pipeline_in_ss(use_default_config=True)\n  File \"/vagrant/src/archivematica/src/dashboard/src/installer/steps.py\", line 108, in setup_pipeline_in_ss\n    api_key=api_key.key,\n  File \"/usr/lib/archivematica/archivematicaCommon/storageService.py\", line 131, in create_pipeline\n    response = _storage_api_session().post(url, json=pipeline)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/requests/sessions.py\", line 590, in post\n    return self.request('POST', url, data=data, json=json, **kwargs)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/requests/sessions.py\", line 542, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/requests/sessions.py\", line 655, in send\n    r = adapter.send(request, **kwargs)\n  File \"/usr/lib/archivematica/archivematicaCommon/storageService.py\", line 75, in send\n    return super(HTTPAdapterWithTimeout, self).send(*args, **kwargs)\n  File \"/usr/share/archivematica/virtualenvs/archivematica/lib/python3.6/site-packages/requests/adapters.py\", line 504, in send\n    raise ConnectTimeout(e, request=request)\nrequests.exceptions.ConnectTimeout: HTTPConnectionPool(host='192.168.168.198', port=8000): Max retries exceeded with url: /api/v2/pipeline/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f241b4b4a90>, 'Connection to 192.168.168.198 timed out. (connect timeout=20.0)'))\n", "path": "/usr/share/archivematica/virtualenvs/archivematica/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin", "syspath": ["/tmp/ansible_django_manage_payload_SdzUWt/ansible_django_manage_payload.zip", "/usr/lib/python2.7", "/usr/lib/python2.7/plat-x86_64-linux-gnu", "/usr/lib/python2.7/lib-tk", "/usr/lib/python2.7/lib-old", "/usr/lib/python2.7/lib-dynload", "/usr/local/lib/python2.7/dist-packages", "/usr/lib/python2.7/dist-packages"]}

PLAY RECAP *********************************************************************
am-local                   : ok=220  changed=141  unreachable=0    failed=1    skipped=187  rescued=0    ignored=5   

```

That is the default IP address for am-local.  I guess the idea is that I have to configure first, and then change the address to something that can be accessed on the LAN?

```
archivematica@westbengal:~/archivematica-1.13.0/deploy-pub/playbooks/archivematica-bionic$ git diff
diff --git a/playbooks/archivematica-bionic/Vagrantfile b/playbooks/archivematica-bionic/Vagrantfile
index f8f0ff0..e4c6fbd 100644
--- a/playbooks/archivematica-bionic/Vagrantfile
+++ b/playbooks/archivematica-bionic/Vagrantfile
@@ -7,10 +7,11 @@ VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = ENV.fetch("VAGRANT_BOX", "ubuntu/bionic64")
+  config.vm.box_version = "20210623.0.0"
 
   {
     "am-local" => {
-      "ip" => "192.168.168.198",
+      "ip" => "192.168.142.134",
       "memory" => "4096",
       "cpus" => "2",
     },
@@ -18,7 +19,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     # Define guest
     config.vm.define short_name do |host|
-      host.vm.network "private_network", ip: properties.fetch("ip")
+      host.vm.network "public_network", ip: properties.fetch("ip")
       host.vm.hostname = "#{short_name}.myapp.dev"
     end
 
archivematica@westbengal:~/archivematica-1.13.0/deploy-pub/playbooks/archivematica-bionic$ 
```

**Comment by hstgelais-crkn-rcdr** [comment]
I went throught the Quick Step documentation and I was able to create some DIP and AIP from a SIP created in the Transfert tab. I deleted those DIP and SIP in the Packages tab in the Storage Service dashboard. 

I even created a DIP and AIP from files located on the nas at /media/crkn-nas-wip/Archivematica after going throught the creation of a SIP on the Transfert tab. 

Access to the Archivematica was given to Beth's team.
